app/src/main/java/app/revanced/integrations/settings/SettingsEnum.java

```diff
@@ -19,7 +19,10 @@ public enum SettingsEnum {
 
     //Video Settings
     OLD_STYLE_QUALITY_SETTINGS("revanced_use_old_style_quality_settings", true, ReturnType.BOOLEAN),
-    PREFERRED_VIDEO_SPEED("revanced_pref_video_speed", -2.0f, ReturnType.FLOAT),
+    //DEFAULT_VIDEO_SPEED("revanced_pref_video_speed", -2.0f, SharedPrefHelper.SharedPrefNames.REVANCED_PREFS, ReturnType.FLOAT),
+    //CUSTOM_PLAYBACK_SPEED_ENABLED("revanced_pref_custom_playback_speed", false, ReturnType.BOOLEAN, true),
+    DEFAULT_VIDEO_QUALITY_WIFI("revanced_pref_video_quality_wifi", -2, SharedPrefHelper.SharedPrefNames.REVANCED_PREFS, ReturnType.INTEGER),
+    DEFAULT_VIDEO_QUALITY_MOBILE("revanced_pref_video_quality_mobile", -2, SharedPrefHelper.SharedPrefNames.REVANCED_PREFS, ReturnType.INTEGER),
```

Comment:
```suggestion
    DEFAULT_VIDEO_QUALITY_WIFI("revanced_default_video_quality_wifi", -2, SharedPrefHelper.SharedPrefNames.REVANCED_PREFS, ReturnType.INTEGER),
    DEFAULT_VIDEO_QUALITY_MOBILE("revanced_default_video_quality_mobile", -2, SharedPrefHelper.SharedPrefNames.REVANCED_PREFS, ReturnType.INTEGER),
```

Make sure to also mirror the changes on the patch PR.